File: Python/PascalsTriangle2_119.py
from typing import List
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        row=[1]
        for i in range(rowIndex):
            next_row=[0]* (len(row)+1)
            for j in range(len(row)):
                next_row[j]+=row[j]
                next_row[j+1]=row[j]
            row=next_row
        return row

s=Solution()

n=int(input("Number of test cases :"))
for i in range(n):
    r=int(input("Row index : "))
    print(s.getRow(r))

File: LeetCode/Java/PascalsTriangle2_119.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

class Solution {
    public List<Integer> getRow(int rowIndex) {
        List<Integer> row =new ArrayList<>();
        row.add(1);
        for(int i=0;i<rowIndex;i++){
            List<Integer> nextRow =new ArrayList<>();
            nextRow.add(1);
            for(int j=1;j<row.size();j++){
                nextRow.add(row.get(j-1)+row.get(j));
            }
            nextRow.add(1);
            row=nextRow;
        }
        return row;
        
    }
}
public class PascalsTriangle2_119 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        Solution sol = new Solution();
        System.out.print("Enter number of test cases: ");
        int n= sc.nextInt();
        sc.nextLine();
        for(int i=0;i<n;i++){
            System.out.print("Number of rows : ");
            int r =sc.nextInt();
            sc.nextLine();
            List<Integer> result = sol.getRow(r);
            System.out.println(result);
        }
    }
}

